LondonBritishLibrary/orient/BLorient801.xml

```diff
@@ -30,22 +31,76 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
                             <idno>Wright 367</idno>
-                            
+
                         </altIdentifier>
-                        
+
                     </msIdentifier>
                     <msContents>
                         <summary/>
-                        
-                        
-                        
-                    </msContents>
-                    
-                    
-                    
-                    
-                    
-                    
+
+<msItem xml:id="ms_i1">
+<locus from="2r"/>
+<title type="complete" ref="LIT1395Fethan"/>
+</msItem>
+
+                        <msItem xml:id="ms_i1.1">
+                            <locus from="2r"/>
+                            <title type="complete" ref="LIT1395Fethan#Introduction"/>
+                            <textLang mainLang="gez"/>
+
+                        </msItem>
+                        <msItem xml:id="ms_i1.2">
+                            <locus from="8v"/>
+                            <title type="complete" ref="LIT1395Fethan#Index"/>
+                            <textLang mainLang="gez"/>
+
+                        </msItem>
+
+                        <msItem xml:id="ms_i1.3">
+                            <locus from="9v"/>
+                            <title type="complete" ref="LIT1395Fethan#Part1"/>
+                            <textLang mainLang="gez"/>
+
+                        </msItem>
+                        <msItem xml:id="ms_i1.4">
+                            <locus from="103v"/>
+                            <title type="complete" ref="LIT1395Fethan#Part2"/>
+                            <textLang mainLang="gez"/>
+
+                        </msItem>
+                        <msItem xml:id="ms_i1.5">
+                            <locus from="224v"/>
+                            <title type="complete" ref="LIT1395Fethan#Appendix"/>
+                            <textLang mainLang="gez"/>
+
+<colophon xml:id="coloph1" xml:lang="gez">
+<locus target="#229v"/>
+መልአ፡ ጽሕፈተ፡ ዝንቱ፡ መጽሐፈ፡ ቀኖና፡ ዘፍትሐ፡ ኵነኔ፡ ሕግ፡ መሲሓዊት፡ በዕለተ፡ ሰንበት፡ አመ፡ ዓሡሩ፡ ለወርኃ፡ መስከረም፡
+በ<date when="0164" calendar="diocletian">፲፻፷ወ፬</date>ዓመተ፡ ሰማዕታት፡ ንጹሓን፡ ቀዳሚ፡ ውስተ፡
+ ምድረ፡ ግብጽ። ወካዕበ፡ ተጽሐፈ፡ ውስተ፡ ብሔርነ፡ ኢትዮጵያ፡ <date calendar="world" when="7181">በ፸፻፡ ወ፩፻፹፡ ወ፩ዓመተ፡ ዓለም</date>።
+ ወእምዓመተ፡ ሥጋዌ፡ <date calendar="ethiopian" when="1681">በ፲፻ወ፯፻፹፡ ወ፩</date>።
+ ወበ<date calendar="alexander" when="0145">፲፻፡ ወ፬፻፡ ወ፭ዓመተ፡ ሰማዕታት</date>።
+ ወእምተንበላት፡ <date calendar="islamic" when="1047">በ፲፻፵፡ ወ፯ዓመት</date>። ወተጽሐፈ፡ በዘመነ፡ ማቴዎስ፡ ወንጌላዊ። እንዘ፡ ዓበቅቴ፡ ፀሓይ፡ ፩፡ ወዓበቅቴ፡
+ወርኅ፡ ፯፡ ወበ፯ዓመተ፡ መንግሥቱ፡ ለንጉሠ፡ ሃይማኖት፡ ዘተሰምየ፡ ኢያሱ፡ ወለእግዚአነ፡ አኰቴት፡ እስከ፡ ለዓለም።
+ወዘአዕለዋ፡ ለዛቲ፡ መጽሐፍ። ይቤ፡ ሕፁፅ፡ ወኃጥእ፡ <gap reason="omitted"/>ወዘአጽሐፈኒ፡ ፍሥሓ፡ ክርስቶስ። ነዳየ፡
```

Comment:
this gap is due to the cataloguer, so type="ellipsis" resp="PRS10171Wright"

Reply:
type causes alarm, so I put reason